Stripe/STPConnectAccountParams.m

```diff
@@ -8,31 +8,54 @@
 
 #import "STPConnectAccountParams.h"
 
-#import "STPLegalEntityParams.h"
-
 NS_ASSUME_NONNULL_BEGIN
 
 @implementation STPConnectAccountParams
 
 @synthesize additionalAPIParameters;
 
 - (instancetype)initWithTosShownAndAccepted:(BOOL)wasAccepted
```

Comment:
since we're changing these signatures anyway, any reason to even have the `wasAccepted` argument?

Reply:
I think we need it to differentiate between if the customer accepted the ToS or if the user didn't show a ToS (ie YES vs nil).